src/pages/ThorChainLP/components/RemoveLiquidity/RemoveLiquidityInput.tsx

```diff
@@ -87,28 +106,131 @@ export const RemoveLiquidityInput: React.FC<RemoveLiquidityProps> = ({
   }, [foundPoolAsset])
 
   const rune = useAppSelector(state => selectAssetById(state, thorchainAssetId))
+  const runeAsset = useAppSelector(state => selectAssetById(state, thorchainAssetId))
 
   const [poolAsset, setPoolAsset] = useState<Asset | undefined>(foundPoolAsset)
   const [userlpData, setUserlpData] = useState<UserLpDataPosition | undefined>()
+  const [runeAccountId, setRuneAccountId] = useState<AccountId | undefined>()
   const [percentageSelection, setPercentageSelection] = useState<number>(50)
+  const [shareOfPoolDecimalPercent, setShareOfPoolDecimalPercent] = useState<string | undefined>()
```

Comment:
Not using this anywhere for remove liquidity

Reply:
<img width="346" alt="Screenshot 2024-02-21 at 9 15 57 am" src="https://github.com/shapeshift/web/assets/97164662/ec080523-af4d-4465-bb1a-e31b1b98b3db">

We _do_ have it on the reusable LP confirm component (available in the PR above this). I'm not sure how useful it is, but it's there for now.